docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -887,7 +887,11 @@ Set to `true` to disable the signout link in the side menu. This is useful if yo
 
 ### signout_redirect_url
 
-URL to redirect the user to after they sign out.
+URL to redirect the user to after they sign out. To support [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) , user need to add `post_logout_redirect_uri` to the `signout_redirect_url`.
+
+Eg. signout_redirect_url = http://localhost:8087/realms/grafana/protocol/openid-connect/logout?post_logout_redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Flogin
+
+> **Note**: `post_logout_redirect_uri` parameter needs to be given after the openid-connect logout url.
```

Comment:
```suggestion
{{% admonition type="note" %}}
The `post_logout_redirect_uri` parameter must be given after the openid-connect logout url.
{{% /admonition %}}
```

Reply:
What do we mean by the word "given"?

Reply:
I meant that post_logout_redirect_uri should be put after the logout url.